public/index.html

```diff
@@ -283,6 +283,11 @@
             <div class="project-name">Netflix Landing Page Clone</div>
             <a href="Netflix_Cloning/Index.html" class="demo-link">Demo</a>
         </div>
+         <div class="table">
+            <div class="day-number">Day 51</div>
+            <div class="project-name">ClimaCode</div>
```

Comment:
i recently merged a PR you add it as 52 
![Uploading image.png…]()

Reply:
![image](https://github.com/dhairyagothi/50_days_50_web_project/assets/142989448/8eb076fb-887f-4da3-9252-7d30560c8bf7)